dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/OperationsParamsValidator.java

```diff
@@ -186,11 +191,14 @@ public TrackedEntity validateTrackedEntity(String trackedEntityUid, User user)
       return null;
     }
 
-    TrackedEntity trackedEntity = trackedEntityService.getTrackedEntity(trackedEntityUid);
+    // TODO(tracker) Are these validations enough? Should we check for ownership too?
```

Comment:
We should probably unify this method with the one present in RelationshipOperationParamsMapper when solving this TODO